assets/js/components.js

```diff
@@ -47,6 +47,27 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      data: 'attributes.environments',
+      createdCell: function (td, _cellData, rowData) {
+        const environments = rowData.attributes.environments
+        const prodEnvironments = environments ? environments.filter(env => env.name === 'prod') : []
+
+        let prodSlackChannel = ''
+        if (prodEnvironments.length === 0) {
+          prodSlackChannel = `N/A`
+        } else {
+          prodSlackChannel = prodEnvironments.map(env => {
+            if (env.alerts_slack_channel === null) {
+              return `Missing slack channel data`
+            } else {
+              return `<a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ env.alerts_slack_channel }}">${env.alerts_slack_channel}</a>`
```

Comment:
I dont think you can create links from the slack channel name - it has to be the channel ID.

Reply:
What Matt said

Reply:
Removed hyperlink